addon/adapters/application.js

```diff
@@ -128,7 +128,6 @@ export default Ember.Object.extend(FetchMixin, Evented, {
     // use resource's service if in container, otherwise use this service to fetch
     let owner = (typeof getOwner === 'function') ? getOwner(this) : this.container;
     let service = owner.lookup('service:' + pluralize(type)) || this;
-    url = this.fetchUrl(url);
```

Comment:
@aars I'm pretty sure this is the hook I use to be sure that when a proxy server is in use the url in the payload for the relationship is replaced with the url of the proxy server.